integrations/huggingface/integration.definition.ts

```diff
@@ -0,0 +1,64 @@
+import { z, IntegrationDefinition, interfaces } from "@botpress/sdk";
+import { integrationName } from "./package.json";
+
+export default new IntegrationDefinition({
+  name: integrationName,
+  version: "0.0.1",
+  readme: "hub.md",
+  icon: "icon.svg",
+  configuration: {
+    schema: z.object({
+      accessToken: z.string(),
+      languageModels: z.string().optional().describe("LLMs to use"),
+    }),
+  },
+  entities: {
+    modelRef: {
+      schema: z.object({
+        id: z.string().describe("Model to use for content generation"),
```

Comment:
**suggestion:** This is where you would list available models normally, but I assume this is the desired behavior since there's so much models on huggingface right ?

Reply:
Exactly, we add only llms, that user adds in the integration configuration as such: https://i.imgur.com/2gx0wnB.png